components/photoview/photobargroup.js

```diff
@@ -100,16 +101,16 @@ const styles = StyleSheet.create({
     position: 'absolute',
     left: '40%',
     bottom: 0,
-    borderRightColor: 'black',
-    borderRightWidth: 1,
+    //borderRightColor: '#eeeeee',
+    //borderRightWidth: 1,
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
 
   },
 
   groupIcon : {
-    backgroundColor: '#F35F64',
+    backgroundColor: 'rgba(220, 220, 220, 0)',
```

Comment:
the reason I changed it was because the height shows up really high which makes the group look off. Also, right now the group isn't clickable so it conveys the wrong impression that it's a button when it's not.

Reply:
discuss later**